### PR TITLE
Implement preconnect to Google fonts

### PR DESCRIPTION
**Pull request by leobaiano**
Implements preconnect to google sources in accordance with issue #105 

My user in WordPress.org is leobaiano.

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -134,6 +134,8 @@ function twentyseventeen_fonts_url() {
 	return esc_url_raw( $fonts_url );
 }
 
+
+
```
Comment (grappler) [inline functions.php]: This is not related.

File: functions.php [hunk 2]
```diff
@@ -240,6 +242,24 @@ function twentyseventeen_scripts() {
...
+ * Add preconnect for Google Fonts.
+ *
+ * @since Twenty Seventeen 1.0
+ *
+ * @param array   $urls           URLs to print for resource hints.
+ * @param string  $relation_type  The relation type the URLs are printed.
+ * @return array  $urls           URLs to print for resource hints.
+ */
+function twentyseventeen_resource_hints( $urls, $relation_type ) {
+    // Add preconnect for Google Fonts if queue twentyfourteen-lato.
```
Comment (grappler) [inline functions.php]: Please use tabs here instead of spaces.

File: functions.php [hunk 3]
```diff
@@ -240,6 +242,24 @@ function twentyseventeen_scripts() {
...
+	if ( wp_style_is( 'twentyseventeen_fonts_url', 'queue' ) && 'preconnect' === $relation_type )
```
Comment (grappler) [inline functions.php]: We need the opening and closing brackets for the if statement.

**Review by grappler — CHANGES_REQUESTED** [review]

File: functions.php (hunk 1 above)
Comment (leobaiano) [reply functions.php]: I had given space to create the function there, when I decided to change place forgot to delete the spaces, thanks for the warning.

Comment (leobaiano) [reply functions.php]: I removed the lines.

File: functions.php (hunk 2 above)
Comment (leobaiano) [reply functions.php]: Done.

File: functions.php (hunk 3 above)
Comment (leobaiano) [reply functions.php]: Done.

**Comment by swissspidy** [comment]
Related: https://core.trac.wordpress.org/ticket/37171

**Comment by leobaiano** [comment]
@swissspidy I pointed out that the issue want to open, see #105 😄

**Comment by davidakennedy** [comment]
Refreshed in #491.

**Pull request closed without merging**